src/config/validators/index.ts

```diff
@@ -64,4 +64,15 @@ export const VALIDATOR_COMMUNITY = [
       cere: ['6SpjH8swCtFwmntQmdikMWyDKgr59q1cLauePYc2iqwwe6Bv'],
     },
   },
+  {
+    name: 'OranG3cluB',
+    Thumbnail: OranG3cluB,
```

Comment:
@Vasya-kripto Could you please provide an image for your Thumbnail: OranG3cluB? I would greatly appreciate your assistance in this matter. Thank you very much for your help.

Reply:
![logo](https://github.com/Cerebellum-Network/staking-dashboard/assets/35801878/9ae1e803-4189-4f56-bb55-7f8821c82cf7)